api/src/groups/test_groups.py

```diff
@@ -5,993 +5,816 @@
-    create_multiple_people(auth_client.sqla, 4)
-    create_multiple_members(auth_client.sqla, 4)
-
-    members = auth_client.sqla.query(Member).all()
 
-    # WHEN member is changed to active
-    for member in members:
-        resp = auth_client.put(url_for('groups.deactivate_member', member_id=member.id))
-        print(member)
-        # THEN assert group is active
-        assert resp.status_code == 200
-        assert resp.json['active'] == False
+@pytest.mark.smoke
+def test_read_all_attendances(auth_client):
+    # GIVEN an empty database
+    # WHEN we read all attendance
+    create_multiple_meetings(auth_client.sqla, random.randint(3, 6))
+    create_multiple_people(auth_client.sqla, random.randint(3, 6))
+    create_multiple_attendance(auth_client.sqla, fraction=0.75)
+    all_attendance = auth_client.sqla.query(Attendance).all()
+    first_meeting = auth_client.sqla.query(Meeting).first()
+    first_person = auth_client.sqla.query(Person).first()
+    # WHEN read all the peron's id and whih meeting they are at
+    # THEN find the attendance in a meeting wich specific meeting id
+    count = 0
+    for attendance in all_attendance:
+        if attendance.meeting_id == first_meeting.id:
+            count+=1
+    resp = auth_client.get(url_for('groups.read_all_attendances', meeting_id = first_meeting.id))
+    print(len(resp.json))
+    # THEN expect the same number of attendacne
+    assert len(resp.json) == count
+# 
+# @pytest.mark.smoke
+# def test_read_attendance_by_member(auth_client):
+#     pass
+# 
+# @pytest.mark.smoke
+# def test_read_attendance_by_meeting(auth_client):
+#     pass
+# 
+@pytest.mark.smoke
+def test_delete_attendance(auth_client):
+    # GIVEN an empty database
+    # WHEN we add in attendance
+    create_multiple_meetings(auth_client.sqla, random.randint(3, 6))
+    create_multiple_people(auth_client.sqla, random.randint(3, 6))
+    first_meeting = auth_client.sqla.query(Meeting).first()
+    first_person = auth_client.sqla.query(Person).first()
+    # WHEN add attendance with perosn_id and meeting_id
+    auth_client.sqla.add(Attendance(meeting_id = first_meeting.id, person_id = first_person.id))
+    auth_client.sqla.commit()
+    # WHEN delete the first meeting
+    resp = auth_client.delete(url_for('groups.delete_attendance', meeting_id = first_meeting.id, person_id = first_person.id))
+    #THEN expect correct message
+    assert resp.status_code == 204
 
-    resp = auth_client.put(url_for('groups.activate_member', member_id=500))
-    # THEN assert group is not found
+    # WHEN delete the not-exist attendance
+    first_person = auth_client.sqla.query(Person).first()
+    resp = auth_client.delete(url_for('groups.delete_attendance', meeting_id = first_meeting.id, person_id = first_person.id))
+    # THEN expect error message
     assert resp.status_code == 404
 
+# 
+# @pytest.mark.smoke
+# def test_repr_group(auth_client):
+#     pass
+# 
+# @pytest.mark.smoke
```

Comment:
This group of unimplemented tests should be marked with the `skip` decorator so that the fact that they're not being run gets reported when we run all the tests. That way we won't forget to go back and implement the tests.